### PR TITLE
Re render

### LINKED ISSUE #115: Rasterio 1.0.21 may need rebuild
I've tried writing this issue 3 or so times now trying to figure out exactly what's going on. If I install rasterio in an existing environment (on OSX currently) I get 1.0.13. If I create a new environment (python=3.7 rasterio) then I get 1.0.21. If I go back to the existing environment and try to install rasterio=1.0.21 I get:

```
The following NEW packages will be INSTALLED:

  postgresql         conda-forge/osx-64::postgresql-10.6-ha1bbaa7_1000
  tzcode             conda-forge/osx-64::tzcode-2018g-h1de35cc_1001

The following packages will be UPDATED:

  libgdal               pkgs/main::libgdal-2.3.3-h0950a36_0 --> conda-forge::libgdal-2.4.0-h89caebc_1002
  rasterio                            1.0.13-py37h54a03ab_0 --> 1.0.21-py37h3683dd5_0

The following packages will be DOWNGRADED:

  cryptography                           2.5-py37hc2b1221_1 --> 2.5-py37hdbc3d79_1
  curl                                    7.64.0-heae2a1f_2 --> 7.64.0-heae2a1f_0
  krb5                                 1.16.3-hcfa6398_1001 --> 1.16.3-h24a3359_1000
  libcurl                                 7.64.0-he376013_2 --> 7.64.0-h76de61e_0
  libpq                                     11.2-h29c6f28_0 --> 10.6-hbe1e24e_1000
  libssh2                               1.8.0-hb1dc21d_1004 --> 1.8.0-hf30b1f0_1003
  openssl                                 1.1.1b-h1de35cc_1 --> 1.0.2r-h1de35cc_0
  python                                3.7.1-hbdd33cc_1002 --> 3.7.1-h145921a_1000

```

It seems that 1.0.21 has only been built for an older build of python (at least conda says "DOWNGRADE") or maybe openssl. Does this mean that 1.0.21 needs to be rebuilt?

### PR DESCRIPTION
**Pull request by ocefpaf**
Should fix #115

### CODE REVIEW

**Comment by djhoese** [comment]
@ocefpaf Do the packages produced from PRs get placed anywhere that I could *prove* that they work in my environments? Like a super hidden conda channel that I could point to (`-c` style).

**Comment by ocefpaf** [comment]
> @ocefpaf Do the packages produced from PRs get placed anywhere that I could _prove_ that they work in my environments? Like a super hidden conda channel that I could point to (`-c` style).

Nope. We need to inspect the logs and see if the right deps are in place. I'll give it a closer look tomorrow morning.

**Comment by ocefpaf** [comment]
No go. It is still downloading `openssl 1.0.2p h1de35cc_1002` b/c it did pull the latest build of gdal :-(

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ requirements:
     - cython
     - numpy
     - libgdal
+    - openssl
```
Comment (ocefpaf) [inline recipe/meta.yaml]: We don't want to merge with this but if this fails we know that the stack is unsolvable at the moment, if it passes then we are between a rock and a hard place b/c can can solve either way and we don't really know what to do to force the right `libgdal` with the latest `openssl` here :-(

Comment (ocefpaf) [reply recipe/meta.yaml]: OK. This exposes the problem on Windows and shows that latest openssl is solvable on OS X. Let me try something else now. Hold on...

**Comment by ocefpaf** [comment]
For some odd reason this is happening in the logs now:

```
The following packages will be UPDATED:

openssl-1.1.1a-h1de35cc_0 conda-forge --> 1.1.1b-h1de35cc_1 conda-forge
```

but the right `libgdal` (`2.4.0-h947adae_1003 conda-forge` built with `openssl-1.1.1a-h1de35cc_0`) is selected :-/

Let's merge this and see if that fixes the `openssl` for you but I'm not sure why `openssl 1.1.1b` is being used even though cb3 should've restricted libgdal to the version (1.1.1a) used to build it.

**Pull request merged**